app/controllers/api/console.php

```diff
@@ -111,3 +115,29 @@
 
         $response->chunk('', true);
     });
+
+App::get('v1/console/resources/:resourceId')
+    ->desc('Check resource ID availability')
+    ->groups(['api', 'projects'])
+    ->label('scope', 'projects.read')
```

Comment:
Lets check with Christy scope for this endpoint. Maybe `sites.write` instead? (because we do it in that flow)